packages/app/pages/login.tsx

```diff
@@ -39,7 +39,16 @@ export default function Login(): ReactElement {
           }
           return Promise.reject(error);
         } else {
-          Router.push(`/api/v1/login/entry?token=${data.token}`);
+          const lastVisited = localStorage.getItem("lastVisited");
+          localStorage.removeItem("lastVisited");
```

Comment:
What if localStorage does't contain `lastVisited` entry? Should we remove it only in the `if (lastVisited) {` block?